README.md

```diff
@@ -8,12 +8,27 @@ This repository contains the assets required to build the [Kubernetes website an
 
 See the [official Hugo documentation](https://gohugo.io/getting-started/installing/) for Hugo installation instructions. Make sure to install the Hugo extended version specified by the `HUGO_VERSION` environment variable in the [`netlify.toml`](netlify.toml#L10) file.
 
-To run the website locally when you have Hugo installed:
+Before building the site, clone the Kubernetes website repository:
 
 ```bash
 git clone https://github.com/kubernetes/website.git
 cd website
 git submodule update --init --recursive
```

Comment:
```suggestion
git submodule update --init --recursive --depth 1
```

(to match the command below)
?

Reply:
Yes, I thought about it. Is there any situation when this is limiting?

Reply:
I think it's fine, we're not expecting to ask contributors to hack on Docsy and its dependencies.